pkg/kubelet/kubelet.go

```diff
@@ -1554,6 +1555,10 @@ func (kl *Kubelet) syncLoop(updates <-chan PodUpdate, handler SyncHandler) {
 			unsyncedPod = true
 		case <-time.After(kl.resyncInterval):
 			glog.V(4).Infof("Periodic sync")
+		case <-syncStatusTicker:
+			glog.V(4).Infof("Periodic status sync")
+			kl.syncStatus()
+			continue
```

Comment:
I'm not sure that it's the best option because with tickers we can accumulate the "signals" to SyncPods() method over time (if SyncPods() will take a lot of time), which I don't think it's good.
But let's wait for others opinions on it.